hvac/v1/__init__.py

```diff
@@ -1181,7 +1181,7 @@ def transit_generate_hmac(self, name, hmac_input, key_version=None, algorithm=No
         return self._post(url, json=params).json()
 
     def transit_sign_data(self, name, input_data, key_version=None, algorithm=None, context=None, prehashed=None,
-                          mount_point='transit'):
+                          mount_point='transit', sign_algorithm='pss'):
```

Comment:
I didn't catch this previously, but since the Vault API parameter is named "signature_algorithm", would you be okay using the same name for these method's for consistency?